src/event_form/steps/event-form-step4.controller.js

```diff
@@ -148,7 +148,22 @@ function EventFormStep4Controller(
    * - on the same location
    */
   function duplicateSearchConditions(data) {
-    return searchApiSwitcher.getDuplicateSearchConditions(data);
+    var location = data.getLocation();
+    if (data.isEvent) {
+      /*jshint camelcase: false*/
```

Comment:
Do you really need these jshint comments?

Reply:
I copied them over from the original location. Not sure how relevant they are.

Reply:
I checked and they're no longer relevant. Removed them in 0ec8bf3.
I believe they were also copy-pasted around in the original location, since [this](https://github.com/cultuurnet/udb3-angular/pull/539/commits/7754615190d5e1df00bf254c084c4f955d1521fa#diff-3a5ab304f1fb6467e0311b99dccd4c2fL58) was the only line it was relevant (but it was also used around other lines).